tutorials/classical_simulation/Simulating with SPAM errors.ipynb

```diff
@@ -0,0 +1,335 @@
+{
```

Comment:
<p><strong>In contrast to</strong> the previous case, when the number of false negatives increases, the system behaves as if all atoms <strong>were </strong>in the ground state leading to a zero population in the excited state <em>for ϵ′=1.</em></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>